packages/app/src/settings/SettingsContainer.vue

```diff
@@ -76,6 +80,23 @@ const props = defineProps<{
   gql: SettingsContainerFragment
 }>()
 
+const route = useRoute()
+
+onMounted(() => {
```

Comment:
We may want to use route watcher here. We don't have it yet, but we probably want a keep-alive around the child views of the non-Spec Runner pages, which would mean that `onMounted` won't run except for the first time the settings page is visited. 

It's also just a little more natural to me that something driven by the route query would re-evaluate when the query is changed, without the component needing to remount.

Reply:
Implemented the route watcher! I still needed the `onMounted` to check that certain elements are rendered since we use `document.getElementById` but now the scrolling itself is determined by watched for route changes.